src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedSimpleProgramEntryPointSymbol.cs

```diff
@@ -33,20 +33,29 @@ internal SynthesizedSimpleProgramEntryPointSymbol(SimpleProgramNamedTypeSymbol c
             _declaration = declaration;
 
             bool hasAwait = declaration.HasAwaitExpressions;
+            bool hasReturnWithExpression = declaration.HasReturnWithExpression;
 
-            if (hasAwait)
+            switch (hasAwait, hasReturnWithExpression)
```

Comment:
>switch [](start = 12, length = 6)

Consider writing as a switch expression.

Reply:
> Consider writing as a switch expression.

I like switch statement better.

---
In reply to: [410423869](https://github.com/dotnet/roslyn/pull/43451#discussion_r410423869) [](ancestors = 410423869)